### PR TITLE
M5G-500 Messaging style polish

### PR DESCRIPTION
**Pull request by cofauver**
# Jira: [M5G-500](https://clever.atlassian.net/browse/M5G-500)

# Overview:
This PR addresses two styling problems that emerged this week.

1. The attachments in quoted announcements were narrower than we wanted them to be.
image

2. The error messages below announcements were not contained below the announcement bubble.
image

I resolved 1. by removing a margin for QuotedAnnouncements.

I resolved 2. by placing a container around the error message and adding some positioning info to the classes and styling them appropriately. 

# Screenshots/GIFs:
### Teacher desktop
image

### Teacher mobile
image

### Student desktop
image

### Student mobile
image

### Desktop
image
### Mobile
image
#### Desktop
image

#### Mobile
image

# Testing:

- [ ] Unit tests
- Manual tests:
  - [x] Chrome
  - [x] Safari
  - [ ] IE11

# Roll Out:

- Before merging:
  - [ ] Updated docs
  - [x] Bumped version in `package.json`

- After merging:
  - [ ] Deployed updated docs (`make deploy-docs`)
  - [ ] Posted in #eng if I made a breaking change to a beta component

### CODE REVIEW

File: src/MessagingThreadHistory/MessageMetadata.tsx [hunk 1]
```diff
@@ -26,16 +26,25 @@ export const MessageMetadata: React.FC<
...
-      {errorMsg && formErrorContainer(errorMsg)}
+      {errorMsg && formErrorContainer(errorMsg, placement)}
     </div>
   );
 });
 
-function formErrorContainer(errorMsg: React.ReactNode): JSX.Element {
+function formErrorContainer(
+  errorMsg: React.ReactNode,
+  placement: "left" | "right" | "center" | "fullWidth",
```
Comment (clcaelynn-clever) [inline src/MessagingThreadHistory/MessageMetadata.tsx:36]: Do we use this same enum elsewhere? Might be good to make it its own type if so.

**Review by clcaelynn-clever — APPROVED** [review]
So sorry this took so long, got lost in the hectic shuffle of today. My bad! 🙁 

Looks good to me, so approving! That said, without having to put them again in the original post, could you check all your screenshot use cases after resolving conflicts/pulling in other clever-components versions? Thank you! 🙌🏻

File: src/MessagingThreadHistory/MessageMetadata.tsx (hunk 1 above)
Comment (cofauver) [reply src/MessagingThreadHistory/MessageMetadata.tsx]: addressed in this commit: https://github.com/Clever/components/pull/677/commits/aa5609a3fdb5a705d256a9b7e6575e94f0327292

**Comment by clcaelynn-clever** [comment]
As far as I can tell, this PR remains good-to-go. 🙌🏻 

@arsalansufi, I believe you have the honor of merging!

**Pull request merged**